extensions/map-layers-formats/src/Icons/MapInfoIcon.ts

```diff
@@ -0,0 +1,9 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+/** @internal */
+export const mapInfoIcon = {
```

Comment:
why isnt this just a string?

Reply:
The required icon doesn't exist in `@bentley/icons-generic-webfont` and there was some discussion about it in the previous PR: https://github.com/iTwin/itwinjs-core/pull/6559#discussion_r1541949326